tests/regression/facebook/test_chrome_facebook_ail_click_photo_viewer_right_arrow.sikuli/test_chrome_facebook_ail_click_photo_viewer_right_arrow.py

```diff
@@ -0,0 +1,39 @@
+# if you are putting your test script folders under {git project folder}/tests/, it will work fine.
+# otherwise, you either add it to system path before you run or hard coded it in here.
+sys.path.append(sys.argv[2])
+import os
+import common
+import facebook
+import shutil
+import browser
+import time
+
+# Disable Sikuli action and info log
+com = common.General()
+com.infolog_enable(0)
+
+chrome = browser.Chrome()
+fb = facebook.facebook()
+
+chrome.clickBar()
+chrome.enterLink(sys.argv[3])
+wait(fb.comment_icons, 15)
```

Comment:
for `wait(fb.comment_icons)` and `hover(fb.comment_icons.targetOffset(-340, -40)`
It exists in 2 files (this case with 2 different browsers). Do you want to make them functions?